website/examples/stress.js

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+
+import {Areas, Brush, Chart, Lines, Points} from 'orama'
+
+const MS_IN_DAY = 24 * 60 * 60 * 1000
+const TODAY = +Date.now()
+
+function getRandomTimeseries(n) {
+  const data = []
+  data.push({
+    date: new Date(+TODAY - n * MS_IN_DAY),
+    value: Math.random() * 50 + 50,
+  })
+  for (let i = 1; i < n; i += 1) {
+    const prev = data[i - 1]
+    data.push({
+      date: new Date(+prev.date + MS_IN_DAY),
+      value: prev.value * (Math.random() * 0.04 + 0.98),
+    })
+  }
+  return data
+}
+
+const data = getRandomTimeseries(5000)
+
+class Stress extends React.Component {
+  state = {
+    xDomain: [data[4900].date, data[4999].date],
+    showBrush: false,
+  }
+
+  handleBrush = ({xDomain}) => this.setState({xDomain})
+
+  toggleBrush = event => this.setState({showBrush: event.target.checked})
+
+  render() {
+    const {xDomain, showBrush} = this.state
+    const filteredData = showBrush
+      ? data.filter(({date}) => (xDomain ? +date >= +xDomain[0] && +date <= xDomain[1] : true))
+      : data
+    return (
+      <React.Fragment>
+        <div>
+          <input type="checkbox" value={this.state.showBrush} onChange={this.toggleBrush} />
+          <label>Show Brush</label>
+        </div>
+        <Chart height={showBrush ? 151 : 211}>
+          <Lines data={filteredData} x="date" y="value" strokeValue="steelblue" interpolate />
+          <Points data={filteredData} x="date" y="value" fillValue="steelblue" />
+        </Chart>
+        {showBrush && (
+          <Brush onUpdate={this.handleBrush} xDomain={xDomain} fillAlphaValue={0.2}>
+            <Chart
+              xShowLabel={false}
+              yShowTicks={false}
+              proportion={0.06}
+              yName=" "
+              xShowGuides={false}
+              yShowGuides={false}
+              x1Name="Start Date"
+              x2Name="End Date"
+            >
+              <Areas data={data} alphaValue={0.3} showHover={false} x="date" />
+              <Lines data={data} lineWidthValue={1} showHover={false} x="date" y="value" />
+            </Chart>
+          </Brush>
+        )}
+      </React.Fragment>
+    )
+  }
+}
```

Comment:
Can we split this into two examples: one with a brush, and one without?

Reply:
Done!